### DISCUSSION TITLE
"Stuck at 'exporting...' while trying to download trained model to iPhone SE2 with Ultralytics Hub iOS"

### DISCUSSION THREAD

**Discussion by kakuteki** [ultralytics/ultralytics#3760] [Q&A]
Hello,

I am using Ultralytics Hub on chrome to train a model in Google Colab. Now, I want to download this trained model to my iPhone SE2, where I have Ultralytics Hub iOS installed.

However, during the download process, the model gets stuck in the "exporting..." state and doesn't complete. I'm looking for assistance in resolving this issue and successfully downloading the trained model to my iPhone SE2 using Ultralytics Hub iOS.

Development Environment:

Ultralytics Hub: Windows 11
Ultralytics Hub iOS: iPhone SE2
Training: Google Colab

![Ultralytics HUB - Google Chrome 2023_07_16 16_17_27](https://github.com/ultralytics/ultralytics/assets/61647657/5928b553-a744-4723-a183-2e670d5c88f0)
![S__21864451](https://github.com/ultralytics/ultralytics/assets/61647657/0c6e61bd-8477-403f-be4e-5a204b681ead)

**Comment by glenn-jocher** [comment]
@kakuteki hello,

Thank you for reaching out with your issue regarding the Ultralytics Hub iOS export process. It seems like you're encountering a hang-up during the model export phase. Here are a few steps you can take to troubleshoot this problem:

1. **Check Network Connection**: Ensure that your iPhone SE2 has a stable internet connection. Network interruptions can cause the export process to stall.

2. **Restart the App**: Sometimes, simply restarting the Ultralytics Hub iOS app can resolve temporary glitches.

3. **Update the App**: Make sure that you are using the latest version of the Ultralytics Hub iOS app, as updates often contain bug fixes and performance improvements.

4. **Clear Cache**: Try clearing the app's cache, as a corrupted cache can sometimes lead to issues with app functionality.

5. **Check Server Status**: Occasionally, server-side issues can affect the export process. You can check if there are any known issues with the Ultralytics Hub that might be causing the problem.

6. **Export Format Compatibility**: Ensure that the export format you are choosing is compatible with the Ultralytics Hub iOS app.

If these steps do not resolve the issue, please visit our documentation at https://docs.ultralytics.com for further support and consider reaching out with a detailed report, including any error messages you might be receiving. Our team is committed to providing assistance and improving your experience with our tools.
